.dotnet/api/api.md

```diff
@@ -840,22 +862,46 @@ namespace OpenAI.Batch {
     }
 }
 namespace OpenAI.Chat {
+    public class AssistantChatMessage : ChatMessage {
```

Comment:
In your description you said "Class inheritance is lost", but I see `AssistantChatMessage` deriving from `ChatMessage` here. Did you mean something different?

Reply:
Ah, you're right -- most are there. The collection types and e.g. struct IEquatables were what I had seen, but also the least problematic.

E.g. this:
```
public readonly struct RunStepStatus : IEquatable<RunStepStatus> {
```

Became:

```
public sealed class RunStepStatus {
```

And this:

```
    public class EmbeddingCollection : ReadOnlyCollection<Embedding> {
        public string Model { get; }
        public EmbeddingTokenUsage Usage { get; }
    }
```

Became:

```
    public class EmbeddingCollection {
        public string Model { get; }
        public EmbeddingTokenUsage Usage { get; }
    }
```